docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3
+FROM ruby:3.4-rc
```

Comment:
The [3.4-rc tag](https://hub.docker.com/layers/library/ruby/3.4-rc/images/sha256-2020ff71b278ee9a10ad6ccddac3f5f5af5eb4fb78ec4fffff92dba00c265d3c) looks outdated. The [3.4 tag](https://hub.docker.com/layers/library/ruby/3.4/images/sha256-f59bb18575803215ad87d96d2e3187fb8bd9d28aa33e3db28803a03de7093895) might be  better choice for staying with the latest 3.4.x series.